sites/main-site/src/content/events/2025/hackathon-march-2025/san-francisco-bay-area.mdx

```diff
@@ -10,10 +10,15 @@ endTime: "17:00-08:00"
 locations:
     - name: To Be Determined
       address: |
-          To Be Determined
+          Amazon | Downtown San Francisco
+          SFO28
+          525 Market St
+          San Francisco 94105
       links:
           - https://nf-co.re/events/2025/hackathon-march-2025
-      geoCoordinates: [37.773972, -122.431297]
+          - https://downtownsf.org/go/amazon
+          - https://seanome.org/
```

Comment:
Can we add a link to Bits and Bio too?

Reply:
What's the involvement / relationship with BiB? Would be nice to mention that in the main text if possible.